package.json

```diff
@@ -79,5 +79,6 @@
   },
   "bugs": {
     "url": "https://github.com/mongodb/leafygreen-ui/issues"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
why this version? `1.22.22` is the latest

Reply:
i think thats whats on my OS lol - the codemod added this

Reply:
Lets add a `^` to the version number to be safe

Reply:
that syntax is not allowed